pkg/extensions/unpack_test.go

```diff
@@ -275,7 +275,8 @@ spec:
       - delete
   title: Sample Crossplane Extension
   version: 0.0.1
-status: {}
+status:
+  conditionedStatus: {}
```

Comment:
This is probably a dumb question, but what is `conditionedStatus`?

Reply:
I believe this first came up here: https://github.com/crossplaneio/crossplane/issues/495#issuecomment-495319702

The conditioned status is a list of independent statuses each focused on a particular condition..
```
status:
  conditionedStatus:
    - type: Raining
      status: false
    - type: Sunny
      status: true
```

https://github.com/crossplaneio/crossplane/blob/db468475610f4f83ddc8a1b51e76837f79b6f52d/pkg/apis/core/v1alpha1/condition.go#L56-L97